src/main/resources/twilio-go/api.mustache

```diff
@@ -128,5 +73,119 @@ func (c *ApiService) {{{nickname}}}({{#allParams}}{{#required}}{{paramName}} {{{
     return nil
 {{/returnType}}
 }
+{{/vendorExtensions.x-is-list-operation}}
+{{#vendorExtensions.x-is-list-operation}}
+// Retrieve a single page of {{{vendorExtensions.x-domain-name}}} records from the API. Request is executed immediately.
+func (c *ApiService) Page{{{vendorExtensions.x-domain-name}}}({{#allParams}}{{#required}}{{paramName}} {{{dataType}}}{{^last}}, {{/last}}{{/required}}{{/allParams}}params *{{{nickname}}}Params, pageToken string, pageNumber string) (*{{{returnType}}}, error) {
+    path := "{{{path}}}"
+
+    {{>partial_serialization}}
+    if pageToken != "" {
+        data.Set("PageToken", pageToken)
+    }
+    if pageToken != "" {
+        data.Set("Page", pageNumber)
+    }
+
+    resp, err := c.requestHandler.Get(c.baseURL+path, data, headers)
+    if err != nil {
+        return nil, err
+    }
+
+    defer resp.Body.Close()
+
+    ps := &{{{returnType}}}{}
+    if err := json.NewDecoder(resp.Body).Decode(ps); err != nil {
+        return nil, err
+    }
+
+    return ps, err
+}
+
+// Lists {{{vendorExtensions.x-domain-name}}} records from the API as a list. Unlike stream, this operation is eager and loads 'limit' records into memory before returning.
+func (c *ApiService) List{{{vendorExtensions.x-domain-name}}}({{#allParams}}{{#required}}{{paramName}} {{{dataType}}}{{^last}}, {{/last}}{{/required}}{{/allParams}}params *{{{nickname}}}Params, limit int) ([]{{{vendorExtensions.x-payload-model-name}}}, error) {
```

Comment:
Left a comment on twilio-go PR, but limit should not be a required param, can we add it to the ListXxxParams struct?